src/localize/languages/pt-PT.json

```
@@ -0,0 +1,470 @@
+          "size": "Tamanho das miniaturas em pixels"
+        },
+        "timeline": {
+          "editor_label": "Controles de linha do tempo",
+          "mode": "Modo",
+          "modes": {
+            "above": "Por cima",
+            "below": "Abaixo",
+            "none": "Nenhum"
+          }
+        },
+        "title": {
+          "duration_seconds": "Segundos para exibir o pop-up (0 = para sempre)",
+          "editor_label": "Editor de titulos",
+          "mode": "Modo de exibição de título de mídia",
+          "modes": {
+            "none": "Sem exibição de título",
+            "popup-bottom-left": "Pop-up no canto inferior esquerdo",
+            "popup-bottom-right": "Pop-up no canto inferior direito",
+            "popup-top-left": "Pop-up no canto superior esquerdo",
+            "popup-top-right": "Pop-up no canto superior direito"
+          }
+        }
+      },
+      "layout": {
+        "fit": "Fit",
+        "fits": {
+          "contain": "Conter",
+          "cover": "Tapar",
+          "fill": "Preencher"
+        },
+        "position": {
+          "x": "Percentagem da localização horizontal",
+          "y": "Percentagem da localização vertical"
+        }
+      },
+      "media_action_conditions": {
+        "all": "Todas as oportunidades",
+        "hidden": "Ao ocultar o navegador/aba",
+        "never": "Nunca",
+        "selected": "Ao selecionar",
+        "unselected": "Ao desselecionar",
+        "visible": "Ao mostrar o navegador/aba"
+      },
+      "timeline": {
+        "clustering_threshold": "A contagem de eventos nos quais eles são agrupados (0 = sem agrupamento)",
+        "media": "A mídia que a linha do tempo exibe",
+        "medias": {
+          "all": "Todos os tipos de mídia",
+          "clips": "Clipes",
+          "snapshots": "Instantâneos"
+        },
+        "show_recordings": "Mostrar gravações",
+        "window_seconds": "A duração padrão da visualização da linha do tempo em segundos"
+      }
+    },
+    "dimensions": {
+      "aspect_ratio": "Proporção padrão (e.g. '16:9')",
+      "aspect_ratio_mode": "Modo de proporção",
+      "aspect_ratio_modes": {
+        "dynamic": "A proporção se ajusta à mídia",
+        "static": "Proporção estática",
+        "unconstrained": "Proporção irrestrita"
+      }
+    },
+    "image": {
+      "layout": "Layout",
+      "mode": "Modo de visualização de imagem",
+      "modes": {
+        "camera": "Instantâneo da câmera do Home Assistant, da entidade de câmera",
+        "screensaver": "Logo Frigate embutido",
+        "url": "Imagem arbitrária especificada por URL"
+      },
+      "refresh_seconds": "Número de segundos após o qual atualizar (0 = nunca)",
+      "url": "Imagem arbitrária especificada por URL"
+    },
+    "live": {
+      "auto_mute": "Silenciar câmeras ao vivo automaticamente",
+      "auto_pause": "Pausar câmeras ao vivo automaticamente",
```

Brazilian expression not used in pt_PT
```suggestion
      "auto_pause": "Parar câmeras ao vivo automaticamente",
```